techniques/developing-localizable-manifests.en.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html dir="ltr" lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Developing Localizable Manifests</title>
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" async="" class="remove"></script>
+    <script class="remove">
+      var respecConfig = {
+          specStatus: "ED",
+	      edDraftURI: "https://w3c.github.io/i18n-drafts/techniques/developing-localizable-manifests.en.html",
```

Comment:
Will this be an [article](https://www.w3.org/International/articlelist) or a document on /TR (like string-meta and specdev)?

Reply:
It's a good question. I used Respec with the idea that it would probably become a note, but I used the i18n-drafts repo instead of creating a new, separate, repo. The other "techniques" documents are actually indexes, so this doesn't really fit as well as I thought it might.

Reply:
This doesn't belong in the drafts/techniques directory, for sure.  It should probably have its own repo, so that we can publish it as a Note in the future.